README.md

```diff
@@ -1,65 +1,102 @@
+<p align="center"><img src="https://github.com/jetstack/cert-manager/blob/master/logo/logo.png" width="250x" /></p>
+</a>
+<a href="https://godoc.org/github.com/cert-manager/istio-csr"><img src="https://godoc.org/github.com/cert-manager/istio-csr?status.svg"></a>
+<a href="https://goreportcard.com/report/github.com/cert-manager/istio-csr"><img alt="Go Report Card" src="https://goreportcard.com/badge/github.com/cert-manager/istio-csr" /></a></p>
+
 # istio-csr
 
-cert-manager-istio-csr is an agent which allows for [istio](https://istio.io) workload
-and control plane components to be secured using
+istio-csr is an agent that allows for [istio](https://istio.io) workload and
+control plane components to be secured using
 [cert-manager](https://cert-manager.io). Certificates facilitating mTLS, inter
 and intra cluster, will be signed, delivered and renewed using [cert-manager
 issuers](https://cert-manager.io/docs/concepts/issuer).
 
 ⚠️ Currently supports istio versions v1.7+
+
 ⚠️ Currently supports cert-manager versions v1.3+
 
 ---
 
 ## Installation
 
+### Installing cert-manager
+
 Firstly, [cert-manager must be
-installed](https://cert-manager.io/docs/installation/) in your cluster. An
-issuer must be configured, which will be used to sign your certificate
-workloads, as well a ready Certificate to serve istiod. Example Issuer and
-istiod Certificate configuration can be found in
+installed](https://cert-manager.io/docs/installation/) in your cluster.
+
+### Issuer or ClusterIssuer
+An Issuer or ClusterIssuer must be configured which will be used to sign istio
+certificates against.
+
+Example Issuer configuration can be found in
 [`./hack/demo/cert-manager-bootstrap-resources.yaml`](./hack/demo/cert-manager-bootstrap-resources.yaml).
+It is advised to not use the CA Issuer type in production environments.
 
-Next, install the cert-manager-istio-csr into the cluster, configured to use
-the Issuer deployed. The Issuer must reside in the same namespace as that
-configured by `-c, --certificate-namespace`, which is `istio-system` by default.
+> It is important to use an issuer type that is able to sign istio mTLS workload
+> certificates (SPIFFE URI SANs) and
+> [istiod serving certificates](./deploy/charts/istio-csr/templates/certificate.yaml).
+> ACME issuers will not work.
 
-```bash
+### Installing istio-csr
+
+Next, install istio-csr into the cluster, configured to use the Issuer or
+ClusterIssuer installed earlier. If using an Issuer rather than the
```

Comment:
Suggestion: add a link as to how to configure istio-csr to use a particular Issuer/ClusterIssuer